src/__tests__/components/login.test.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
```

Comment:
Remove the __tests__ directory. This directory does not exist in any other branch.